Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -30,17 +30,6 @@ public final class ProjectPamphletViewController: UIViewController {
       |> \.translatesAutoresizingMaskIntoConstraints .~ false
   }()
 
-  private let backThisProjectContainerSublayer: CAShapeLayer = {
```

Comment:
We don't need this `CAShapeLayer` anymore because we can set all these properties on the `backThisProjectContainerView`'s layer and use `maskedCorners` instead.